inference_engine/src/controller.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+import numpy as np
+from json import dumps
+
+from src.config import Config
+from .storage import Storage
+from .preprocessing import Preprocessing
+from .model import AnswerModel, IndexModel
+from .dto import CheckExamsDTO, CheckExamDTO, GenerateExamKeyDTO
+
+
+class Controller:
+    @staticmethod
+    def check_exam(request: CheckExamDTO):
+        return Controller._mark_detection(request.exam_path, request.exam_name)
```

Comment:
@marekhering  I'm not exactly sure what PEP is specifying here but you are returning output of _mark_detection which does not return anything

Reply:
PEP said that return is unnecessary here, it is already done